main.go

```diff
@@ -60,8 +61,8 @@ func main() {
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", true,
 		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
-	flag.StringVar(&logFile, "log-file", "", "The path to a file to write logs to.")
```

Comment:
Why are these two flags being removed?

Reply:
Good catch, these changes where from something different, seems like I mixed this up.